inital commit to StarPixie branch

ElectriPixie opened the pull request.

ElectriPixie commented:
Merged Initial commit

aussig commented:
@ElectriPixie I see you put in this PR for some PowerPlay tracking, but have now closed it and deleted your repo. Could you let me know the status of your work and what the objective is/was?

ElectriPixie commented:
Oh hey. Sorry for the late reply. I actually was setting up merit tracking,
I have adhd and it's really difficult for me to integrate with the
reporting requirements to fly with my squadron. Sorry I didn't see this
sooner. I haven't checked my email in a while. I got the actual stuff into
the interface and got the tracking working but the way they write the
merits to the journal file is a pain to deal with. So I shelved it even
though I could maybe still make the mod into a useful tool for others, it's
not as useful for me like it is, and I stopped flying lately because I'm
busy filling my portfolio with stuff for my job search. I was just hoping I
could make a quick mod to help us all with the paperwork to keep track of
what we are all doing, and maybe have another repo for my portfolio in the
end so I could share it with the elite dangerous community.

On Tue, Dec 17, 2024 at 3:26 AM Aussi ***@***.***> wrote:

> @ElectriPixie <https://github.com/ElectriPixie> I see you put in this PR
> for some PowerPlay tracking, but have now closed it and deleted your repo.
> Could you let me know the status of your work and what the objective is/was?
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/aussig/BGS-Tally/pull/284#issuecomment-2547788347>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AFMRKJVDPWOD46H3HR3NTS32F7NZVAVCNFSM6AAAAABTXEXXV6VHI2DSMVQWIX3LMV43OSLTON2WKQ3PNVWWK3TUHMZDKNBXG44DQMZUG4>
> .
> You are receiving this because you were mentioned.Message ID:
> ***@***.***>
>

aussig commented:
> Oh hey. Sorry for the late reply. I actually was setting up merit tracking, I have adhd and it's really difficult for me to integrate with the reporting requirements to fly with my squadron. Sorry I didn't see this sooner. I haven't checked my email in a while. I got the actual stuff into the interface and got the tracking working but the way they write the merits to the journal file is a pain to deal with. So I shelved it even though I could maybe still make the mod into a useful tool for others, it's not as useful for me like it is, and I stopped flying lately because I'm busy filling my portfolio with stuff for my job search. I was just hoping I could make a quick mod to help us all with the paperwork to keep track of what we are all doing, and maybe have another repo for my portfolio in the end so I could share it with the elite dangerous community.
> […](#)
> On Tue, Dec 17, 2024 at 3:26 AM Aussi ***@***.***> wrote: @ElectriPixie <https://github.com/ElectriPixie> I see you put in this PR for some PowerPlay tracking, but have now closed it and deleted your repo. Could you let me know the status of your work and what the objective is/was? — Reply to this email directly, view it on GitHub <[#284 (comment)](https://github.com/aussig/BGS-Tally/pull/284#issuecomment-2547788347)>, or unsubscribe <https://github.com/notifications/unsubscribe-auth/AFMRKJVDPWOD46H3HR3NTS32F7NZVAVCNFSM6AAAAABTXEXXV6VHI2DSMVQWIX3LMV43OSLTON2WKQ3PNVWWK3TUHMZDKNBXG44DQMZUG4> . You are receiving this because you were mentioned.Message ID: ***@***.***>

Ah yeah, no worries. There's not enough in the journal right now to track merits in a sensible 'live' way while playing the game. Hopefully FD will give the journal some love, it desperately needs it (in other areas too).

ElectriPixie commented:
You are welcome to the code I did write to get the stuff into the
interface, it just won't work the way I intended, unless they write the
journal more than once on start.

On Fri, Feb 14, 2025 at 10:37 AM Electric Pixie ***@***.***>
wrote:

> Oh hey. Sorry for the late reply. I actually was setting up merit
> tracking, I have adhd and it's really difficult for me to integrate with
> the reporting requirements to fly with my squadron. Sorry I didn't see this
> sooner. I haven't checked my email in a while. I got the actual stuff into
> the interface and got the tracking working but the way they write the
> merits to the journal file is a pain to deal with. So I shelved it even
> though I could maybe still make the mod into a useful tool for others, it's
> not as useful for me like it is, and I stopped flying lately because I'm
> busy filling my portfolio with stuff for my job search. I was just hoping I
> could make a quick mod to help us all with the paperwork to keep track of
> what we are all doing, and maybe have another repo for my portfolio in the
> end so I could share it with the elite dangerous community.
>
> On Tue, Dec 17, 2024 at 3:26 AM Aussi ***@***.***> wrote:
>
>> @ElectriPixie <https://github.com/ElectriPixie> I see you put in this PR
>> for some PowerPlay tracking, but have now closed it and deleted your repo.
>> Could you let me know the status of your work and what the objective is/was?
>>
>> —
>> Reply to this email directly, view it on GitHub
>> <https://github.com/aussig/BGS-Tally/pull/284#issuecomment-2547788347>,
>> or unsubscribe
>> <https://github.com/notifications/unsubscribe-auth/AFMRKJVDPWOD46H3HR3NTS32F7NZVAVCNFSM6AAAAABTXEXXV6VHI2DSMVQWIX3LMV43OSLTON2WKQ3PNVWWK3TUHMZDKNBXG44DQMZUG4>
>> .
>> You are receiving this because you were mentioned.Message ID:
>> ***@***.***>
>>
>

aussig commented:
> You are welcome to the code I did write to get the stuff into the interface, it just won't work the way I intended, unless they write the journal more than once on start. On Fri, Feb 14, 2025 at 10:37 AM Electric Pixie ***@***.***> wrote:
> […](#)
> Oh hey. Sorry for the late reply. I actually was setting up merit tracking, I have adhd and it's really difficult for me to integrate with the reporting requirements to fly with my squadron. Sorry I didn't see this sooner. I haven't checked my email in a while. I got the actual stuff into the interface and got the tracking working but the way they write the merits to the journal file is a pain to deal with. So I shelved it even though I could maybe still make the mod into a useful tool for others, it's not as useful for me like it is, and I stopped flying lately because I'm busy filling my portfolio with stuff for my job search. I was just hoping I could make a quick mod to help us all with the paperwork to keep track of what we are all doing, and maybe have another repo for my portfolio in the end so I could share it with the elite dangerous community. On Tue, Dec 17, 2024 at 3:26 AM Aussi ***@***.***> wrote: > @ElectriPixie <https://github.com/ElectriPixie> I see you put in this PR > for some PowerPlay tracking, but have now closed it and deleted your repo. > Could you let me know the status of your work and what the objective is/was? > > — > Reply to this email directly, view it on GitHub > <[#284 (comment)](https://github.com/aussig/BGS-Tally/pull/284#issuecomment-2547788347)>, > or unsubscribe > <https://github.com/notifications/unsubscribe-auth/AFMRKJVDPWOD46H3HR3NTS32F7NZVAVCNFSM6AAAAABTXEXXV6VHI2DSMVQWIX3LMV43OSLTON2WKQ3PNVWWK3TUHMZDKNBXG44DQMZUG4> > . > You are receiving this because you were mentioned.Message ID: > ***@***.***> >

I can't - you deleted your repository a few minutes after creating the PR and long before I had a chance to review it.  This automatically closes the PR and stops it being able to be merged.

ElectriPixie commented:
I've never made a PR for someone else's project on github to be honest. I
can set one up just so you have the code available, I just have to go
google what to do. I'll set it back up for the experience, it's probably
good to know how to do it.

On Fri, Feb 14, 2025 at 10:52 AM Aussi ***@***.***> wrote:

> You are welcome to the code I did write to get the stuff into the
> interface, it just won't work the way I intended, unless they write the
> journal more than once on start. On Fri, Feb 14, 2025 at 10:37 AM Electric
> Pixie *@*.
>
> *> wrote: … <#m_-3258554769613362154_> Oh hey. Sorry for the late reply. I
> actually was setting up merit tracking, I have adhd and it's really
> difficult for me to integrate with the reporting requirements to fly with
> my squadron. Sorry I didn't see this sooner. I haven't checked my email in
> a while. I got the actual stuff into the interface and got the tracking
> working but the way they write the merits to the journal file is a pain to
> deal with. So I shelved it even though I could maybe still make the mod
> into a useful tool for others, it's not as useful for me like it is, and I
> stopped flying lately because I'm busy filling my portfolio with stuff for
> my job search. I was just hoping I could make a quick mod to help us all
> with the paperwork to keep track of what we are all doing, and maybe have
> another repo for my portfolio in the end so I could share it with the elite
> dangerous community. On Tue, Dec 17, 2024 at 3:26 AM Aussi @.*> wrote: >
> @ElectriPixie <https://github.com/ElectriPixie>
> https://github.com/ElectriPixie I see you put in this PR > for some
> PowerPlay tracking, but have now closed it and deleted your repo. > Could
> you let me know the status of your work and what the objective is/was? > >
> — > Reply to this email directly, view it on GitHub > <#284 (comment)
> <https://github.com/aussig/BGS-Tally/pull/284#issuecomment-2547788347>>,
> > or unsubscribe >
> https://github.com/notifications/unsubscribe-auth/AFMRKJVDPWOD46H3HR3NTS32F7NZVAVCNFSM6AAAAABTXEXXV6VHI2DSMVQWIX3LMV43OSLTON2WKQ3PNVWWK3TUHMZDKNBXG44DQMZUG4
> > . > You are receiving this because you were mentioned.Message ID: > *@*.***>
> >
>
> I can't - you deleted your repository a few minutes after creating the PR
> and long before I had a chance to review it. This automatically closes the
> PR and stops it being able to be merged.
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/aussig/BGS-Tally/pull/284#issuecomment-2659689181>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AFMRKJXO7JMMOF4HQPMIRXD2PYGNTAVCNFSM6AAAAABTXEXXV6VHI2DSMVQWIX3LMV43OSLTON2WKQ3PNVWWK3TUHMZDMNJZGY4DSMJYGE>
> .
> You are receiving this because you were mentioned.Message ID:
> ***@***.***>
> [image: aussig]*aussig* left a comment (aussig/BGS-Tally#284)
> <https://github.com/aussig/BGS-Tally/pull/284#issuecomment-2659689181>
>
> You are welcome to the code I did write to get the stuff into the
> interface, it just won't work the way I intended, unless they write the
> journal more than once on start. On Fri, Feb 14, 2025 at 10:37 AM Electric
> Pixie *@*.
>
> *> wrote: … <#m_-3258554769613362154_> Oh hey. Sorry for the late reply. I
> actually was setting up merit tracking, I have adhd and it's really
> difficult for me to integrate with the reporting requirements to fly with
> my squadron. Sorry I didn't see this sooner. I haven't checked my email in
> a while. I got the actual stuff into the interface and got the tracking
> working but the way they write the merits to the journal file is a pain to
> deal with. So I shelved it even though I could maybe still make the mod
> into a useful tool for others, it's not as useful for me like it is, and I
> stopped flying lately because I'm busy filling my portfolio with stuff for
> my job search. I was just hoping I could make a quick mod to help us all
> with the paperwork to keep track of what we are all doing, and maybe have
> another repo for my portfolio in the end so I could share it with the elite
> dangerous community. On Tue, Dec 17, 2024 at 3:26 AM Aussi @.*> wrote: >
> @ElectriPixie <https://github.com/ElectriPixie>
> https://github.com/ElectriPixie I see you put in this PR > for some
> PowerPlay tracking, but have now closed it and deleted your repo. > Could
> you let me know the status of your work and what the objective is/was? > >
> — > Reply to this email directly, view it on GitHub > <#284 (comment)
> <https://github.com/aussig/BGS-Tally/pull/284#issuecomment-2547788347>>,
> > or unsubscribe >
> https://github.com/notifications/unsubscribe-auth/AFMRKJVDPWOD46H3HR3NTS32F7NZVAVCNFSM6AAAAABTXEXXV6VHI2DSMVQWIX3LMV43OSLTON2WKQ3PNVWWK3TUHMZDKNBXG44DQMZUG4
> > . > You are receiving this because you were mentioned.Message ID: > *@*.***>
> >
>
> I can't - you deleted your repository a few minutes after creating the PR
> and long before I had a chance to review it. This automatically closes the
> PR and stops it being able to be merged.
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/aussig/BGS-Tally/pull/284#issuecomment-2659689181>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AFMRKJXO7JMMOF4HQPMIRXD2PYGNTAVCNFSM6AAAAABTXEXXV6VHI2DSMVQWIX3LMV43OSLTON2WKQ3PNVWWK3TUHMZDMNJZGY4DSMJYGE>
> .
> You are receiving this because you were mentioned.Message ID:
> ***@***.***>
>

aussig commented:
> I've never made a PR for someone else's project on github to be honest. I can set one up just so you have the code available, I just have to go google what to do. I'll set it back up for the experience, it's probably good to know how to do it.
> […](#)
> On Fri, Feb 14, 2025 at 10:52 AM Aussi ***@***.***> wrote: You are welcome to the code I did write to get the stuff into the interface, it just won't work the way I intended, unless they write the journal more than once on start. On Fri, Feb 14, 2025 at 10:37 AM Electric Pixie *@*. *> wrote: … <#m_-3258554769613362154_> Oh hey. Sorry for the late reply. I actually was setting up merit tracking, I have adhd and it's really difficult for me to integrate with the reporting requirements to fly with my squadron. Sorry I didn't see this sooner. I haven't checked my email in a while. I got the actual stuff into the interface and got the tracking working but the way they write the merits to the journal file is a pain to deal with. So I shelved it even though I could maybe still make the mod into a useful tool for others, it's not as useful for me like it is, and I stopped flying lately because I'm busy filling my portfolio with stuff for my job search. I was just hoping I could make a quick mod to help us all with the paperwork to keep track of what we are all doing, and maybe have another repo for my portfolio in the end so I could share it with the elite dangerous community. On Tue, Dec 17, 2024 at 3:26 AM Aussi @.*> wrote: > @ElectriPixie <https://github.com/ElectriPixie> https://github.com/ElectriPixie I see you put in this PR > for some PowerPlay tracking, but have now closed it and deleted your repo. > Could you let me know the status of your work and what the objective is/was? > > — > Reply to this email directly, view it on GitHub > <#284 (comment) <[#284 (comment)](https://github.com/aussig/BGS-Tally/pull/284#issuecomment-2547788347)>>, > or unsubscribe > https://github.com/notifications/unsubscribe-auth/AFMRKJVDPWOD46H3HR3NTS32F7NZVAVCNFSM6AAAAABTXEXXV6VHI2DSMVQWIX3LMV43OSLTON2WKQ3PNVWWK3TUHMZDKNBXG44DQMZUG4 > . > You are receiving this because you were mentioned.Message ID: > *@*.***> > I can't - you deleted your repository a few minutes after creating the PR and long before I had a chance to review it. This automatically closes the PR and stops it being able to be merged. — Reply to this email directly, view it on GitHub <[#284 (comment)](https://github.com/aussig/BGS-Tally/pull/284#issuecomment-2659689181)>, or unsubscribe <https://github.com/notifications/unsubscribe-auth/AFMRKJXO7JMMOF4HQPMIRXD2PYGNTAVCNFSM6AAAAABTXEXXV6VHI2DSMVQWIX3LMV43OSLTON2WKQ3PNVWWK3TUHMZDMNJZGY4DSMJYGE> . You are receiving this because you were mentioned.Message ID: ***@***.***> [image: aussig]*aussig* left a comment ([aussig/BGS-Tally#284](https://github.com/aussig/BGS-Tally/pull/284)) <[#284 (comment)](https://github.com/aussig/BGS-Tally/pull/284#issuecomment-2659689181)> You are welcome to the code I did write to get the stuff into the interface, it just won't work the way I intended, unless they write the journal more than once on start. On Fri, Feb 14, 2025 at 10:37 AM Electric Pixie *@*. *> wrote: … <#m_-3258554769613362154_> Oh hey. Sorry for the late reply. I actually was setting up merit tracking, I have adhd and it's really difficult for me to integrate with the reporting requirements to fly with my squadron. Sorry I didn't see this sooner. I haven't checked my email in a while. I got the actual stuff into the interface and got the tracking working but the way they write the merits to the journal file is a pain to deal with. So I shelved it even though I could maybe still make the mod into a useful tool for others, it's not as useful for me like it is, and I stopped flying lately because I'm busy filling my portfolio with stuff for my job search. I was just hoping I could make a quick mod to help us all with the paperwork to keep track of what we are all doing, and maybe have another repo for my portfolio in the end so I could share it with the elite dangerous community. On Tue, Dec 17, 2024 at 3:26 AM Aussi @.*> wrote: > @ElectriPixie <https://github.com/ElectriPixie> https://github.com/ElectriPixie I see you put in this PR > for some PowerPlay tracking, but have now closed it and deleted your repo. > Could you let me know the status of your work and what the objective is/was? > > — > Reply to this email directly, view it on GitHub > <#284 (comment) <[#284 (comment)](https://github.com/aussig/BGS-Tally/pull/284#issuecomment-2547788347)>>, > or unsubscribe > https://github.com/notifications/unsubscribe-auth/AFMRKJVDPWOD46H3HR3NTS32F7NZVAVCNFSM6AAAAABTXEXXV6VHI2DSMVQWIX3LMV43OSLTON2WKQ3PNVWWK3TUHMZDKNBXG44DQMZUG4 > . > You are receiving this because you were mentioned.Message ID: > *@*.***> > I can't - you deleted your repository a few minutes after creating the PR and long before I had a chance to review it. This automatically closes the PR and stops it being able to be merged. — Reply to this email directly, view it on GitHub <[#284 (comment)](https://github.com/aussig/BGS-Tally/pull/284#issuecomment-2659689181)>, or unsubscribe <https://github.com/notifications/unsubscribe-auth/AFMRKJXO7JMMOF4HQPMIRXD2PYGNTAVCNFSM6AAAAABTXEXXV6VHI2DSMVQWIX3LMV43OSLTON2WKQ3PNVWWK3TUHMZDMNJZGY4DSMJYGE> . You are receiving this because you were mentioned.Message ID: ***@***.***>

You did the PR fine in December (this conversation is all attached to that PR). It's just that you then immediately deleted your forked repository, which immediately closes the PR.

ElectriPixie commented:
I just added my patches so they can merge with your develop branch, and
made a pull request so you can see what I was working on <3

On Fri, Feb 14, 2025 at 11:01 AM Aussi ***@***.***> wrote:

> I've never made a PR for someone else's project on github to be honest. I
> can set one up just so you have the code available, I just have to go
> google what to do. I'll set it back up for the experience, it's probably
> good to know how to do it.
>
> You did the PR fine in December (this conversation is all attached to that
> PR). It's just that you then immediately deleted your forked repository,
> which immediately closes the PR.
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/aussig/BGS-Tally/pull/284#issuecomment-2659708931>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AFMRKJXDV2YC3CSTS4WXCP32PYHNTAVCNFSM6AAAAABTXEXXV6VHI2DSMVQWIX3LMV43OSLTON2WKQ3PNVWWK3TUHMZDMNJZG4YDQOJTGE>
> .
> You are receiving this because you were mentioned.Message ID:
> ***@***.***>
> [image: aussig]*aussig* left a comment (aussig/BGS-Tally#284)
> <https://github.com/aussig/BGS-Tally/pull/284#issuecomment-2659708931>
>
> I've never made a PR for someone else's project on github to be honest. I
> can set one up just so you have the code available, I just have to go
> google what to do. I'll set it back up for the experience, it's probably
> good to know how to do it.
>
> You did the PR fine in December (this conversation is all attached to that
> PR). It's just that you then immediately deleted your forked repository,
> which immediately closes the PR.
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/aussig/BGS-Tally/pull/284#issuecomment-2659708931>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AFMRKJXDV2YC3CSTS4WXCP32PYHNTAVCNFSM6AAAAABTXEXXV6VHI2DSMVQWIX3LMV43OSLTON2WKQ3PNVWWK3TUHMZDMNJZG4YDQOJTGE>
> .
> You are receiving this because you were mentioned.Message ID:
> ***@***.***>
>

ElectriPixie commented:
If you look at it and think it might be useful to set it up for manual
tracking to integrate it all in one output, then send me a list of what
ever changes it would need for the PR to be accepted and I'll tune up
whatever it is.

On Fri, Feb 14, 2025 at 12:04 PM Electric Pixie ***@***.***>
wrote:

> I just added my patches so they can merge with your develop branch, and
> made a pull request so you can see what I was working on <3
>
> On Fri, Feb 14, 2025 at 11:01 AM Aussi ***@***.***> wrote:
>
>> I've never made a PR for someone else's project on github to be honest. I
>> can set one up just so you have the code available, I just have to go
>> google what to do. I'll set it back up for the experience, it's probably
>> good to know how to do it.
>>
>> You did the PR fine in December (this conversation is all attached to
>> that PR). It's just that you then immediately deleted your forked
>> repository, which immediately closes the PR.
>>
>> —
>> Reply to this email directly, view it on GitHub
>> <https://github.com/aussig/BGS-Tally/pull/284#issuecomment-2659708931>,
>> or unsubscribe
>> <https://github.com/notifications/unsubscribe-auth/AFMRKJXDV2YC3CSTS4WXCP32PYHNTAVCNFSM6AAAAABTXEXXV6VHI2DSMVQWIX3LMV43OSLTON2WKQ3PNVWWK3TUHMZDMNJZG4YDQOJTGE>
>> .
>> You are receiving this because you were mentioned.Message ID:
>> ***@***.***>
>> [image: aussig]*aussig* left a comment (aussig/BGS-Tally#284)
>> <https://github.com/aussig/BGS-Tally/pull/284#issuecomment-2659708931>
>>
>> I've never made a PR for someone else's project on github to be honest. I
>> can set one up just so you have the code available, I just have to go
>> google what to do. I'll set it back up for the experience, it's probably
>> good to know how to do it.
>>
>> You did the PR fine in December (this conversation is all attached to
>> that PR). It's just that you then immediately deleted your forked
>> repository, which immediately closes the PR.
>>
>> —
>> Reply to this email directly, view it on GitHub
>> <https://github.com/aussig/BGS-Tally/pull/284#issuecomment-2659708931>,
>> or unsubscribe
>> <https://github.com/notifications/unsubscribe-auth/AFMRKJXDV2YC3CSTS4WXCP32PYHNTAVCNFSM6AAAAABTXEXXV6VHI2DSMVQWIX3LMV43OSLTON2WKQ3PNVWWK3TUHMZDMNJZG4YDQOJTGE>
>> .
>> You are receiving this because you were mentioned.Message ID:
>> ***@***.***>
>>
>
